app/scripts/modules/amazon/src/serverGroup/details/scalingPolicy/targetTracking/targetTrackingPopover.html

```diff
@@ -0,0 +1,6 @@
+<target-tracking-chart style="height: 150px;"
+                       config="$ctrl.policy.targetTrackingConfiguration"
+                       ticks="{x: 12, y: 5}"
```

Comment:
`target-tracking-chart` and `metric-alarm-chart` both use the same style/layout defaults, maybe pull them out?